composer.json

```
@@ -31,6 +31,6 @@
         "symfony/twig-bundle": "4.1.0",
         "symfony/web-profiler-bundle": "6.1.* <6.1.2",
         "twig/intl-extra": "3.9.0",
-        "twig/twig": "2.7.0 || 3.9.0 || 3.10.*"
+        "twig/twig": "2.7.0 || 3.9.0"
```

```suggestion
        "twig/twig": "2.7.0 || 3.9.0 || 3.10.0 || 3.10.1"
```

The conflicts on `3.10.0` and `3.10.1` need to stay in any case.

---

Correct.